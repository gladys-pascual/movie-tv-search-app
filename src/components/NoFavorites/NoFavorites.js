import React from "react";
import { Link } from "react-router-dom";

const NoFavorites = () => {
  return (
    <>
      <section className="no-shows">
        <span className="material-icons">sentiment_very_dissatisfied</span>
        <h3>You haven't saved your favourite movie or tv shows yet.</h3>
        <h3>
          Search your favourite movies and tv shows using the search bar in our
          {"   "}
          <Link to="/">homepage.</Link>
        </h3>
      </section>
    </>
  );
};

export default NoFavorites;
