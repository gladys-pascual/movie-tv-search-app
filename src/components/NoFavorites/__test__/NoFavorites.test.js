import React from "react";
import { render } from "@testing-library/react";
import NoFavorites from "../NoFavorites";
import { BrowserRouter } from "react-router-dom";

test("renders NoFavorites compinent as expected", () => {
  const { container } = render(
    <BrowserRouter>
      <NoFavorites />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
