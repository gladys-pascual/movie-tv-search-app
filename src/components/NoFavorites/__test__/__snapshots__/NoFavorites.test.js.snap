// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NoFavorites compinent as expected 1`] = `
<div>
  <section
    class="no-shows"
  >
    <span
      class="material-icons"
    >
      sentiment_very_dissatisfied
    </span>
    <h3>
      You haven't saved your favourite movie or tv shows yet.
    </h3>
    <h3>
      Search your favourite movies and tv shows using the search bar in our
         
      <a
        href="/"
      >
        homepage.
      </a>
    </h3>
  </section>
</div>
`;
